include/deal.II/lac/precondition.h

```diff
@@ -2073,6 +2081,21 @@ PreconditionChebyshev<MatrixType,VectorType,PreconditionerType>::estimate_eigenv
   const double alpha = (data.smoothing_range > 1. ?
                         max_eigenvalue / data.smoothing_range :
                         std::min(0.9*max_eigenvalue, min_eigenvalue));
+
+  // in case the user set the degree to invalid unsigned int, we have to
+  // determine the number of necessary iterations from the Chebyshev error
+  // estimate, given the target tolerance specified by smoothing_range. This
+  // estimate is based on the error formula given in section 5.1 of
+  // R. S. Varga, Matrix iterative analysis, 2nd ed., Springer, 2009
```

Comment:
is this reference mentioned in class documentation? If not, i would add it somewhere as a bibtex entry.